select-book-theme.ftd

```diff
@@ -824,7 +824,7 @@ width: fill
 region: h3
 padding-bottom: 12
 boolean open: true
-append-at: inner
+append-at: inner-4
```

Comment:
This is smell. The id has to be unique on the "outside". The ids inside the component are not really unique as there are many possible instance of the same component.